java/vector/src/test/java/org/apache/arrow/vector/pojo/TestConvert.java

```diff
@@ -48,6 +55,73 @@
  */
 public class TestConvert {
 
+  private static String badSchemaJson = "{\n" +
+    "  \"fields\" : [ {\n" +
+    "    \"nullable\" : true,\n" +
+    "    \"type\" : {\n" +
+    "      \"name\" : \"struct\"\n" +
+    "    },\n" +
+    "    \"children\" : [ {\n" +
+    "      \"nullable\" : true,\n" +
+    "      \"type\" : {\n" +
+    "        \"name\" : \"list\"\n" +
+    "      },\n" +
+    "      \"children\" : [ {\n" +
+    "        \"nullable\" : true,\n" +
+    "        \"type\" : {\n" +
+    "          \"name\" : \"null\"\n" +
+    "        },\n" +
+    "        \"children\" : [ ],\n" +
+    "        \"typeLayout\" : {\n" +
+    "          \"vectors\" : [ ]\n" +
+    "        },\n" +
+    "        \"name\" : \"[DEFAULT]\"\n" +
+    "      } ],\n" +
+    "      \"typeLayout\" : {\n" +
+    "        \"vectors\" : [ {\n" +
+    "          \"type\" : \"VALIDITY\",\n" +
+    "          \"typeBitWidth\" : 1\n" +
+    "        }, {\n" +
+    "          \"type\" : \"OFFSET\",\n" +
+    "          \"typeBitWidth\" : 32\n" +
+    "        } ]\n" +
+    "      },\n" +
+    "      \"name\" : \"list\"\n" +
+    "    }, {\n" +
+    "      \"nullable\" : true,\n" +
+    "      \"type\" : {\n" +
+    "        \"name\" : \"fixedsizelist\",\n" +
+    "        \"listSize\" : 5\n" +
+    "      },\n" +
+    "      \"children\" : [ {\n" +
+    "        \"nullable\" : true,\n" +
+    "        \"type\" : {\n" +
+    "          \"name\" : \"null\"\n" +
+    "        },\n" +
+    "        \"children\" : [ ],\n" +
+    "        \"typeLayout\" : {\n" +
+    "          \"vectors\" : [ ]\n" +
+    "        },\n" +
+    "        \"name\" : \"$data$\"\n" +
+    "      } ],\n" +
+    "      \"typeLayout\" : {\n" +
+    "        \"vectors\" : [ {\n" +
+    "          \"type\" : \"VALIDITY\",\n" +
+    "          \"typeBitWidth\" : 1\n" +
+    "        } ]\n" +
+    "      },\n" +
+    "      \"name\" : \"fixedlist\"\n" +
+    "    } ],\n" +
+    "    \"typeLayout\" : {\n" +
+    "      \"vectors\" : [ {\n" +
+    "        \"type\" : \"VALIDITY\",\n" +
+    "        \"typeBitWidth\" : 1\n" +
+    "      } ]\n" +
+    "    },\n" +
+    "    \"name\" : \"a\"\n" +
+    "  } ]\n" +
+    "}\n";
```

Comment:
This is a large block that's very hard to read just to check the name of a field.  Is there a more compact way you can test this?

Reply:
done